Dockerfile

```diff
@@ -50,8 +50,8 @@ FROM debian:10.3-slim as argoexec-base
 
 ARG IMAGE_OS=linux
 
-# NOTE: keep the version synced with https://storage.googleapis.com/kubernetes-release/release/stable.txt
-ENV KUBECTL_VERSION=1.15.1
+# NOTE: kubectl version should be one minor version less than https://storage.googleapis.com/kubernetes-release/release/stable.txt
```

Comment:
Why one minor version less?

Reply:
The Kubernetes project maintains release branches for the most recent three minor releases (1.19, 1.18, 1.17).
Kubernetes 1.19 and newer receive approximately 1 year of patch support.
Kubernetes 1.18 and older received approximately 9 months of patch support.
kubectl is supported within one minor version (older or newer) of kube-apiserver.

Example: kube-apiserver is at 1.18, then kubectl is supported at 1.19, 1.18, and 1.17

Given the above, argo should likely remain on one minor less (1.18) than the current stable (1.19). This allows support for all three active releases of k8s.